src/_backend_agg_wrapper.cpp

```diff
@@ -430,7 +430,7 @@ static PyObject *PyRendererAgg_draw_quad_mesh(PyRendererAgg *self, PyObject *arg
                                       offsets,
                                       offset_trans,
                                       facecolors,
-                                      antialiased,
+                                      antialiased != 0,
```

Comment:
wouldn't it be simpler (throughout) to make antialiased a bool?

Reply:
Done, mostly.